barretenberg/cpp/src/barretenberg/ultra_honk/decider_keys.hpp

```diff
@@ -69,6 +69,22 @@ template <typename Flavor_, size_t NUM_ = 2> struct DeciderProvingKeys_ {
         return results;
     }
 
+    typename Flavor::template ProverUnivariates<2> row_to_short_univariates(size_t row_idx) const
+    {
+        auto prover_polynomials_views = get_polynomials_views();
+        typename Flavor::template ProverUnivariates<2> results;
+        // Set the size corresponding to the number of rows in the execution trace
+        // Iterate over the prover polynomials' views corresponding to each proving key
+        for (size_t dpk_idx = 0; auto& get_all : prover_polynomials_views) {
```

Comment:
get_all can be made `const` I believe and made renamed `view`

Reply:
could you clarify by what you mean by "renamed view"? I don't fully understand

Reply:
sorry "made renamed" was a typo, I meant rename to `view` and make const. But no strong preference to the name, was just thinking that you're iteration over something called views